server/s3_copy_object_action.h

```diff
@@ -38,6 +38,15 @@ enum class S3CopyObjectActionState {
   completed,                // All stages done completely
 };
 
+const u_long MaxCopyObjectSourceSize = 5368709120ULL;  // 5GB
```

Comment:
you can have this as macro defined in cc

Reply:
I avoid using pre-processor macros to define constants, as that is not good coding practice these days.

For reference: https://rules.sonarsource.com/cpp/RSPEC-5028

Do you have something else in mind where macro would be better instead of const expression ?